.changeset/light-cars-fail.md

```diff
@@ -0,0 +1,6 @@
+---
+"comet-site-preview-v1": major
+---
+
+Migration to Open Source
```

Comment:
This information is not relevant

Reply:
I didn't want to leave the changelogs empty. Should i leave them blank?